pyrasterframes/src/main/python/docs/time-series.pymd

```diff
@@ -53,34 +54,30 @@ Now we have a catalog with several months of MODIS data for a single granule. Ho
 
 ```python read_catalog
 raster_cols = ['B01', 'B02',] # red and near-infrared respectively
-park_rf = spark.read.raster(catalog=park_cat.select(['acquisition_date', 'granule_id'] + raster_cols),
-                            catalog_col_names=raster_cols,
-                           ) \
-                    .withColumn('park', st_geomFromWKT(lit(park_geo.wkt)))
-park_rf.createOrReplaceTempView('catalog_read')
+park_rf = spark.read.raster(
+    catalog=park_cat.select(['acquisition_date', 'granule_id'] + raster_cols),
+    catalog_col_names=raster_cols
+    ) \
+    .withColumn('park', st_geomFromWKT(lit(park_geo.wkt)))
 park_rf.printSchema()
+park_rf.persist()                    
 ```
 
 ## Vector and Raster Data Interaction
 
 Now we have the vector representation of the park boundary alongside the _tiles_ of red and near infrared bands. Next, we need to create a _tile_ representation of the park to allow us to limit the time series analysis to pixels within the park. This is similar to the masking operation demonstrated in @ref:[NoData handling](nodata-handling.md#masking).
 
-We do this using two SQL transformations. The first one will reproject the park boundary from coordinates to the MODIS sinusoidal projection. The second one will create a new _tile_ aligned with the imagery containing a value of 1 where the pixels are contained within the park and NoData elsewhere. The code demonstrates that we can use any `rf_` or `st_` function in SQL. For a more thorough demonstration of how Python and SQL codes compare in RasterFrames, see the @ref:[language support section](languages.md).
+We do this using two transformations. The first one will reproject the park boundary from coordinates to the MODIS sinusoidal projection. The second one will create a new _tile_ aligned with the imagery containing a value of 1 where the pixels are contained within the park and NoData elsewhere. 
 
 ```python burn_in
-cr_1 = spark.sql("""
-SELECT *,
-    st_reproject(park, 'EPSG:4326', rf_crs(B01).crsProj4) as park_native
-FROM catalog_read
-""")
-cr_1.createOrReplaceTempView('cr_1')
-
-cr_2 = spark.sql("""
-SELECT *,
-    rf_rasterize(park_native, rf_geometry(B01), 1, rf_dimensions(B01).cols, rf_dimensions(B01).rows) as park_tile
-FROM cr_1
-""").where(rf_tile_sum('park_tile') > 0)
+cr_1 = park_rf.withColumn('park_native', st_reproject('park', lit('EPSG:4326'), rf_crs('B01')))
+
+cr_2 = cr_1 \
+    .withColumn('dims', rf_dimensions('B01')) \
+    .withColumn('park_tile', rf_rasterize('park_native', rf_geometry('B01'), lit(1), 'dims.cols', 'dims.rows')) \
```

Comment:
possibly break this long line up. on narrower aspect ratios it will be easier to read